daphne_worker/src/config.rs

```diff
@@ -645,14 +650,29 @@ impl<'srv> DaphneWorker<'srv> {
     /// `hpke_config_id` is cached (if it exists).
     pub(crate) async fn get_hpke_receiver_config(
         &self,
-        hpke_receiver_kv_key: HpkeReceiverKvKey,
-    ) -> Result<Option<HpkeReceiverConfigKvPair>> {
-        self.kv_get_cached(
-            &self.isolate_state().hpke_receiver_configs,
-            KV_KEY_PREFIX_HPKE_RECEIVER_CONFIG,
-            Cow::Owned(hpke_receiver_kv_key),
-        )
-        .await
+        version: DapVersion,
+        config_id: u8,
+    ) -> Result<Option<HpkeReceiverConfig>> {
+        Ok(self
+            .get_hpke_receiver_configs(version)
+            .await?
+            .and_then(|mut configs| configs.remove(&config_id)))
+    }
+
+    /// Get a reference to the HPKE receiver configs, ensuring that the config indicated by
+    /// `hpke_config_id` is cached (if it exists).
+    pub(crate) async fn get_hpke_receiver_configs(
+        &self,
+        version: DapVersion,
+    ) -> Result<Option<HpkeRecieverConfigSet>> {
+        Ok(self
+            .kv_get_cached(
+                &self.isolate_state().hpke_receiver_configs,
+                KV_KEY_PREFIX_HPKE_RECEIVER_CONFIG,
```

Comment:
I think it makes sense to rename this variable now that this string is no longer a prefix of the KV key. (It's now the KV key itself.) While at it, what do you think about renaming the key to `"hpke_receiver_config_set"`? It's currently `"hpke_receiver_config"`.
```suggestion
                KV_KEY_HPKE_RECEIVER_CONFIG_SET,
```

Reply:
Bump